ext/dapr-ext-fastapi/dapr/ext/fastapi/actor.py

```diff
@@ -48,23 +48,27 @@ def _wrap_response(
 
 
 class DaprActor(object):
-    def __init__(self, app: FastAPI):
-        self._dapr_serializer = DefaultJSONSerializer()
+    _DEFAULT_ACTOR_ROUTER_TAGS: List[str] = ['Actor']
+
+    def __init__(self, app: FastAPI,
+                 router_tags: Optional[List[str]] = None):
+        self._router_tags = router_tags if router_tags else self._DEFAULT_ACTOR_ROUTER_TAGS
         self._router = APIRouter()
+        self._dapr_serializer = DefaultJSONSerializer()
         self.init_routes(self._router)
         app.include_router(self._router)
 
     def init_routes(self, router: APIRouter):
-        @router.get("/healthz")
+        @router.get("/healthz", tags=self._router_tags)
```

Comment:
Sure - though I'm the only maintainer of the SDK so the risk someone forgets is slim :)

I think it goes back to the intent behind this. I figured you want the automatically provided routes to be annotated but anything you implement yourself doesn't need to be annotated.

I think it would be confusing to annotate user-provided handlers (even if they are on an actor). This is why I think only the "magic" methods should be annotated automatically.

Reply:
Thats not what i meant
what i meant is when and if you will add another "magic" method we wouldn't want you to forget and add the tags

Reply:
Yes I understand that part - you want a comment so that a maintainer of this code will not forget to apply the tags to other magic methods.